Gemfile

```diff
@@ -41,7 +41,7 @@ gem 'aws-sdk-kms'
 gem 'aws-sdk-s3', '~> 1'
 gem 'aws-sdk-sns', '~> 1'
 gem 'betamocks', git: 'https://github.com/department-of-veterans-affairs/betamocks', branch: 'master'
-gem 'bgs_ext', git: 'https://github.com/department-of-veterans-affairs/bgs-ext.git', require: 'bgs'
+gem 'bgs_ext', git: 'https://github.com/department-of-veterans-affairs/bgs-ext.git', branch: 'API-15184-ClaimsApi-insertintenttofile-allow-optional-clmantSsn-attribute', require: 'bgs'
```

Comment:
You'll wanna revert this once https://github.com/department-of-veterans-affairs/bgs-ext/pull/74 gets merged

Reply:
Updated [here](https://github.com/department-of-veterans-affairs/vets-api/pull/9810/commits/a1dee3955147e8d42b26c8c651c479c5712c5566).